moped-editor/src/components/GridTable/Search.js

```diff
@@ -206,10 +219,7 @@ const Search = ({
    * Clears the simple search when switching to filters
    */
   const handleSwitchToAdvancedSearch = () => {
-    searchState.setSearchParameters({
-      value: "",
-      column: "",
-    });
```

Comment:
This seemed outdated from a time when the simple search used both a column and a search term value. The columns used in the simple search box are defined in `PROJECT_LIST_VIEW_QUERY_CONFIG.columns` so I updated this with the new state setter defined in `useSearch`.